index.js

```diff
@@ -425,4 +429,18 @@
     return JSON.parse(s);
   });
 
+  //  regexp  /////////////////////////////////////////////////////////////////
```

Comment:
The linter is complaining that this line exceeds the maximum line length by one character. Sorry for the hassle.

I appreciate your responsiveness. I :heart: having you as a contributor, @svozza!